docs/guide/snaps.md

```diff
@@ -30,55 +31,38 @@ Among other things, SES allows us to restrict access to global JavaScript APIs a
 
 ## Getting Started
 
-There are many simple snaps available for you to learn from and begin to understand what your own snap needs. The following tutorial will walk you through a "Hello, World!" snap.
-
 ### Prerequisites
+- [Node.js 16.0](https://nodejs.org/) or later.
 
-- Node.js
-  - We recommend using [nvm](https://github.com/nvm-sh/nvm). If you don't want to use nvm, your local Node version needs to be greater than or equal to the one listed [here](https://github.com/MetaMask/template-snap/blob/main/.nvmrc).
-- git
-- [yarn v1](https://classic.yarnpkg.com)
-- Up-to-date Chromium or Firefox browser
```

Comment:
We should keep those

Reply:
I get what you mean as these are not wrong. But aren't we cutting ourselves / Snaps short by really getting down to the bare basics of what's required? For the users that don't have git, yarn, and Chromium, they'll in the following steps anyhow.

I looked for inspo at some of the popular docs and want to keep it short and to the point.

Reply:
I get what you mean as these are not wrong. But in this case, aren't we selling ourselves short by giving (too much) attention to anything that's not Snaps? For users that don't have basics such as git, yarn, and Chromium, they'll realise that they're required in the following steps.

I looked for inspo at some of the popular docs and want to keep it short and to the point.